### PR TITLE
Add What's New In DevTools (Chrome 79)

### PR DESCRIPTION
**Pull request by kaycebasques**
What's changed, or what was fixed?

- Adds `/src/content/en/updates/2019/10/devtools.md`

**Fixes:** N/A

**Target Live Date:** 2019-10-17

- [ ] This has been reviewed and approved by @kaycebasques 
- [ ] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2019/10/devtools.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Initiator tab {: #initiator }
+
+[Network request initiator stack trace](https://chromium.googlesource.com/chromium/src/+/0d0cae574b458c3d4bd7f6f254f74f6f2442b710)
+
+## Emulate CSS media features
+
+[prefers-color-scheme: dark](https://chromium.googlesource.com/chromium/src/+/9ca30329b8ee53b2462d72772dc189385b6e2a34)
+
+## Code coverage
```
Comment (natorion) [inline src/content/en/updates/2019/10/devtools.md]: I think there were some other minor tweaks being done by @sigurdschneider e.g. clicking on the empty coverage gutter will display the Coverage panel etc.

File: src/content/en/updates/2019/10/devtools.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+### Cookie value previews {: #cookiepreviews }
+
+[Show values when selected](https://chromium.googlesource.com/chromium/src/+/2c327126ab78b37dfbcedee2a8a267d2a54dadfc)
+
+## Initiator tab {: #initiator }
+
+[Network request initiator stack trace](https://chromium.googlesource.com/chromium/src/+/0d0cae574b458c3d4bd7f6f254f74f6f2442b710)
+
+## Emulate CSS media features
```
Comment (natorion) [inline src/content/en/updates/2019/10/devtools.md]: Isn't dark mode also in there already? We are cutting the branch this Thursday after all @mathiasbynens

Comment (mathiasbynens) [reply src/content/en/updates/2019/10/devtools.md]: Yep, this covers dark mode. Dark/light mode is a "CSS media feature". It's just that the functionality is not limited to just dark mode, you can also emulate other such media features, like `prefers-reduced-motion: reduce`.

Comment (kaycebasques) [reply src/content/en/updates/2019/10/devtools.md]: I think `prefers-color-scheme: dark` is the most attention-grabbing headline but I will rework this section to mention the other new features

File: src/content/en/updates/2019/10/devtools.md (hunk 1 above)
Comment (kaycebasques) [reply src/content/en/updates/2019/10/devtools.md]: I'll check the commit log again to see if there's other stuff to cover

Comment (kaycebasques) [reply src/content/en/updates/2019/10/devtools.md]: I mentioned the gutter feature

**Pull request merged**